examples/ex04/ex04b/2BlockFriction_Isoilunit.i

```diff
@@ -0,0 +1,527 @@
+    rank_two_tensor = stress
+    variable = stress_yy
+    index_i = 1
+    index_j = 1
+    [../]
+    [./strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_yy
+    index_i = 1
+    index_j = 1
+    [../]
+    [./stress_zz]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_zz
+    index_i = 2
+    index_j = 2
+    [../]
+    [./strain_zz]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_zz
+    index_i = 2
+    index_j = 2
+    [../]
+    [./stress_xy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_xy
+    index_i = 0
+    index_j = 1
+    [../]
+    [./strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_xy
+    index_i = 0
+    index_j = 1
+    [../]
+    [./stress_yz]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_yz
+    index_i = 1
+    index_j = 2
+    [../]
+    [./strain_yz]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_yz
+    index_i = 1
+    index_j = 2
+    [../]
+    [./accel_x]
+    type = NewmarkAccelAux
+    variable = accel_x
+    displacement = disp_x
+    velocity = vel_x
+    beta = 0.25
+    execute_on = timestep_end
+    [../]
+    [./vel_x]
+    type = NewmarkVelAux
+    variable = vel_x
+    acceleration = accel_x
+    gamma = 0.5
+    execute_on = timestep_end
+    [../]
+    [./accel_y]
+    type = NewmarkAccelAux
+    variable = accel_y
+    displacement = disp_y
+    velocity = vel_y
+    beta = 0.25
+    execute_on = timestep_end
+    [../]
+    [./vel_y]
+    type = NewmarkVelAux
+    variable = vel_y
+    acceleration = accel_y
+    gamma = 0.5
+    execute_on = timestep_end
+    [../]
+    [./accel_z]
+    type = NewmarkAccelAux
+    variable = accel_z
+    displacement = disp_z
+    velocity = vel_z
+    beta = 0.25
+    execute_on = timestep_end
+    [../]
+    [./vel_z]
+    type = NewmarkVelAux
+    variable = vel_z
+    acceleration = accel_z
+    gamma = 0.5
+    execute_on = timestep_end
+    [../]
+    [./layers]
+    type = UniformLayerAuxKernel
+    variable = layer_id
+    interfaces = '11.0'
+    direction = '0.0 0.0 1.0'
+    block = 1002
+    execute_on = initial
+    [../]
+[]
+
+[BCs]
+    [./bottom_x]
+    type = PresetBC
+    variable = disp_x
+    boundary = '109 111 112 113 114'
+    value = 0.0
+    [../]
+    [./bottom_y]
+    type = PresetBC
+    variable = disp_y
+    boundary = '109 111 112 113 114'
+    value = 0.0
+    [../]
+    [./bottom_z]
+    type = PresetBC
+    variable = disp_z
+    boundary = '109 111 112 113 114'
+    value = 0.0
+    [../]
+    [./top_x]
+    type = PresetDisplacement
+    boundary = 502
+    variable = disp_x
+    beta = 0.25
+    velocity = vel_x
+    acceleration = accel_x
+    function = loading_bc
+    [../]
+[]
+
+[Functions]
+    [./loading_bc]
+    type = PiecewiseLinear
+    data_file = 'Displacement2.csv'
```

Comment:
If this file is same as the ones used in other examples, you can provide the path to that file as shown below instead of checking in multiple copies of the same displacement time history. 

`data_file = '../../ex01/Displacement2.csv'`